### ISSUE TITLE
8.0 "geologist" benefit from mining.

### ISSUE THREAD

**Issue by elfl0rd** [StrangeLoopGames/EcoSuggestions#680]
https://github.com/StrangeLoopGames/EcoIssues/issues/11057

Why not, maybe we can have "geologist" benefit in mining?

e.g. mining lvl 3 - you now have geologist lvl 1; you can see hot\cold (https://considerable.com/hot-and-cold/) indicator of same ores with latest mined ore (if you drive excavator\skidsteer or hold any pickaxe on hand) in ~5 blocks range.
mining lvl 5 - you now have geologist lvl 2; in addition to lvl 1 you can see direction (up or down) in ~7-10 blocks range
mining lvl 7 - you now have geologist lvl 3; now you can SEE same ore (one block, closest to you) with latest mined ore, trough another blocks in ~10-12 blocks range.

also this change make mining usefull at late game.

**Comment by Conchipedia** [comment]
maybe have a see in world option to se a rough area of location of where ore might be found, allowing for the fact an ore can be founf roughly 10-20 blocks from this hot spot in any direction

**Comment by elfl0rd** [comment]
> maybe have a see in world option to se a rough area of location of where ore might be found, allowing for the fact an ore can be found roughly 10-20 blocks from this hot spot in any direction

ofc, i think it's another suggestion (maybe issue), need minimap layers for all ores etc. Like species.

**Comment by gardien78** [comment]
Hi, i was wondering, is it sure that ithere is ron ore on my generated world, or is there chance there a glitch in 8.0?
Or its really hard to find any now?

Server is solid vanilla, no mods.
I checked many place on the globe, and i notice a crazy amount of basalt underwater everywhere.
No sign of iron ore or gold ore or copper ore anywhere on the surface.
Even coal seems to be nowhere. Two of my players digged for over 2 Hours strait like crazy and cant find any coal as deep as they go.

cant seem to find anything to confirm or deny this on the forum. Closest i found about it is your discussion here.

Do we need to dig for like 1 day or 2 to get there? Is it normal there no sign of fragment on the surface?

Basically any information you can give me will be greatly appreciated.

Thanks in advance.

**Comment by Conchipedia** [comment]
not sure if minmap could give an accurate idea of ores like species, but we have one for oil where certain areas are regarded as being better, a bit like a heat map saying in certain areas you stand a better chance of finding some more than others

**Comment by jellomold** [comment]
farmers have the soil sampler, miners could have an ore sampler. outputs a 3d graph of the ore bodies in an X by Y by Z frustum in the direction the tool was used

**Comment by Conchipedia** [comment]
but we don't want it to pinpoint **exact** coordinates (as this is not true in real life) , just give a general idea of roughly where to find stuff

**Comment by jellomold** [comment]
the frustum idea can be visualized as "20% iron, 70% sandstone, 10% coal"  from a depth of A to B.  like when you use it, you dial in the depth (or distance along the selected axis) you take the core sample from, and the sampler returns values for the subsequent 10-20 blocks or something maybe tied to mining skill level for depth of sampler, or accuracy.

**Comment by jellomold** [comment]
mining is mostly a 25m3 process with 2x2 hallways connecting

**Comment by jellomold** [comment]
the cyclinder size also potentially limits the effectiveness.. like a 3 block diameter vs a 10 block diameter.. could work the same as 'stone axe, metal axe, modern axe' ...  'basic core sampler', 'deep core sampler', 'spectrographic core sampler'

**Comment by Conchipedia** [comment]
just had to Google this
frustum
nounGeometry
noun: frustum; plural noun: frusta; plural noun: frustums

    the portion of a cone or pyramid which remains after its upper part has been cut off by a plane parallel to its base, or which is intercepted between two such planes

not all of us are clever 😢 

Geologists need to find the ore deposits that are hidden underground. Different geologic processes concentrate mineral resources. They study geologic formations searching for areas that are likely to have ore deposits. They test the physical and chemical properties of soil and rocks. For example, they might test rocks to see if the rocks are magnetic or contain certain chemical elements. Then, geologists make maps of their findings to locate possible ore deposits. Today, satellites do some of the work for geologists. Satellites can make maps of large areas more quickly than geologists on the ground can.

so maybe we have like a tool that will generate a Map like the Environment data that is produced for Animals/plants/oil etc.

i mean i could live with having a map of an area and then go digging in it

**Comment by redwyre** [comment]
It would be interesting to have something like a https://en.wikipedia.org/wiki/Ground-penetrating_radar
Which could require interpretation of the resulting radargram.

**Comment by Kastuk** [comment]
> "20% iron, 70% sandstone, 10% coal"
That's the perfect idea, just like Spectrometer from Mars Colony: Challenger.
Just add one more string to the Soil Sampler that only Miners will see.
